FLEDGE.md

```diff
@@ -419,6 +455,33 @@ The [Product-level TURTLEDOVE](https://github.com/WICG/turtledove/blob/master/PR
+* `browserSignals.ageInMinutes`: How long since the interest group was most recently joined, in minutes, as an integer. Guaranteed to be between 0 and 43200 (the number of minutes in 30 days, the maximum lifetime of an interest group), inclusive.
+* `browserSignals.ageInMinutesMax60`: Same as `browserSignals.ageInMinutes`, but with a maximum value of 60, which is returned if the group is more than an hour old.
+* `browserSignals.ageInHoursMax24`: The interest group's age in hours, as an integer, with a maximum value of 24. Always non-negative.
+* `browserSignals.ageInDaysMax30`: The interest group's age in days, as an integer, with a maximum value of 30. Always non-negative.
+
+If the resulting dot-product is negative, the interest group is immediately removed from an auction (note that if there's no `priorityVector`, interest groups with negative values currently are not filtered from auctions). After calculating new priorities as needed, and filtering out interest groups with negative calculated priorities, the `perBuyerGroupLimits` value is applied to all interest groups of a given owner, unless the interest group's `enableBiddingSignalsPrioritization` field is present and true.
+
+If `enableBiddingSignalsPrioritization` is true, then rather than applying `perBuyerGroupLimits` immediately after the calculating the sparse dot product as desribed above, group limit enforcement is delayed until after fetching the trusted bidding signals. In this case, if the trusted bidding signals specify a per-interest-group `priorityVector` for an interest group, the dot product of that `priorityVector` is again taken with a `prioritySignals` vector. The `prioritySignals` vector is the same as in the first calculation, except that that there's an addition `browserSignals.firstDotProductPriority` value, which is the result of multiplying the interest group's `priorityVector`, if present, with the `prioritySignals` of the auction. If this new dot product is negative, the interest group is removed from the auction. If there is no `priorityVector` for an interest group, the priority from earlier in the auction is used instead. Once all priorities have been calculated, then `perBuyerGroupLimits` is applied, and the auction continues as normal.
+
+if `enableBiddingSignalsPrioritization` is false, then the `priorityVector` from the trusted bidding singals will still be multiplied by the `prioritySignals` as above, but it will only be used to skip the interest group if the result is less than 0. This parameter exists to improve performance - when it's false for all interest groups for a particular bidder in an auction, that bidder's interest groups can be filtered out before any bidding signals are fetched, reducing network usage and server load.
+
+
```

Comment:
This is pretty complicated and I'm wondering if a few little examples might help illustrate how it can be used.
Perhaps how this could be used as a filter to prevent showing an ad on sites about politics (i.e. interest group has "politics: -1" in it's priorityVector and auction config has "poitics: 1" in it's priorityVector).
Perhaps how a [user defined function](https://github.com/privacysandbox/fledge-docs/blob/main/key_value_service_trust_model.md#support-for-user-defined-functions-udfs) could be used in the bidding signals server to filter out an ad by sending back a negative priority.
Perhaps how a priority vector and the ageInDays value could be used to only show an ad during the first 24 hours after joining, i.e. ad with "ageInDays: -1".

Reply:
I've gone ahead and given an example with 3 different IGs.  I don't think we want to invest too much in it, unless we want to significantly rework the doc - it doesn't really give specific examples anywhere else (It does give template-ish examples of structs, but nothing more specific).